app/helpers/application_helper.rb

```diff
@@ -1,2 +1,21 @@
 module ApplicationHelper
+  def nightly_rate_helper
+    [
+    ["<$75", 7500],
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.